lib/data/card.dart

```diff
@@ -15,30 +15,38 @@ enum CardCategory {
 }
 
 class CardData {
-  int id;
-  final CardCategory category;
-  String name;
-  final CardType cardType;
-  final String cardNo;
+  int? id;
+  final CardCategory? category;
+  String? name;
+  final CardType? cardType;
```

Comment:
Shouldn't be nullable

Reply:
There will have a conflict with [code](https://github.com/nfcim/nfsee/blob/master/lib/ui/home.dart#:~:text=if%20(pendingName%20%3D%3D%20%22%22)%20%7B-,data.name%20%3D%20null%3B,-%7D%20else%20%7B) in the home.dart, if we insist that `name` is nullable.

Reply:
`cardType` is related to this [function](https://github.com/nfcim/nfsee/blob/master/lib/data/card.dart#:~:text=final%20cardType%20%3D%20getEnumFromString%3CCardType%3E(CardType.values%2C%20data%5B%27card_type%27%5D)%3B) that may have a null value return.